gnosis/gnosis-spec.ini

```diff
@@ -95,6 +95,36 @@ localMem: M
     andBool SIGS_BUF ==K selectRange(M, SIGS_LOC +Int 32, SIGS_LEN)
 +attribute: [trusted, matching(#gas)]
 
+; external call to signatureSplit should not change the state
+[signatureSplit_external]
+k: (#execute => #halt) ~> _
+output: _ => _
+statusCode: _ => EVMC_SUCCESS
+callStack: _
+this: #PROXY_ID
+msg_sender: MSG_SENDER
+callData: #abiCallData("signatureSplit", (
+            #bytes(#buf(SIGS_LEN, SIGS)),
+            #uint256(POS) ))
```

Comment:
can we make this more general?  like:
```
#parseByteStack(substrString(Keccak256(#generateSignature(FNAME, ARGS)), 0, 8))
```
where FNAME = `"signatureSplit"` and ARGS = `#bytes(_), #uint256(_)`?

Reply:
```
<k>
  JUMPI 408 bool2Word ( 168831172 ==K 4294967295 &Int chop ( #asWord ( 184 : 23 : 220 : 194 : #take ( 28 , WS ) ) /Int (2^224) ) ) ...
</k>
```